.github/workflows/find-all-tools.yml

```diff
@@ -0,0 +1,79 @@
+name: Test All Tools
+
+on:
+  workflow_call:
+    outputs:
+      matrix:
+        description: 'Matrix of tools to test'
+        value: ${{ jobs.find-tools.outputs.matrix }}
+
+permissions:
+  contents: read
+
+jobs:
+  find-tools:
+    name: Find all tools
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          repository: 'nishaq503/image-tools'
+          ref: 'master'
+      - name: Find tools
+        id: set-matrix
+        run: |
+          # List of directories to ignore
+          ignored_dirs="polus-python-template ftl-label .venv"
```

Comment:
does the template tool create issues?

Reply:
Yes. The filename that contains the two `{` and `}` causes problems. Also, the updated tools are found using the presence of any `pyproject.toml` file which conflicts with the template